source/amazon/services/prerequisites/dependencies.rst

```diff
@@ -35,22 +35,36 @@ Pip
 
 The required modules can be installed with Pip, the Python package manager. Most of UNIX distributions have this tool available in their software repositories, but it can be compiled from sources too:
 
-a) For **CentOS/RHEL/Fedora** systems:
+A) For **CentOS/RHEL/Fedora** systems:
+
+a. Using ``yum``:
 
 .. code-block:: console
 
   # yum update && yum install python3-pip
 
-b) For **Debian/Ubuntu** systems:
+b. From sources:
+
+.. code-block:: console
+
+  # yum update && yum install curl python3-distutils
+  # curl -O https://bootstrap.pypa.io/get-pip.py
+  # python3 get-pip.py
+
+
+B) For **Debian/Ubuntu** systems:
+
+a. Using ``apt-get``:
 
 .. code-block:: console
 
   # apt-get update && apt-get install python3-pip
 
-c) From sources:
+b. From sources:
 
 .. code-block:: console
 
+  # apt-get update && apt-get install curl python3-distutils
   # curl -O https://bootstrap.pypa.io/get-pip.py
   # python3 get-pip.py
 
```

Comment:
Same here. I understand the first lines were added for consistency but I would like to know if there is a big difference between these two paths.